src/Algebra/Module/Morphism/Properties.agda

```diff
@@ -0,0 +1,125 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of linear maps.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Module.Morphism.Properties where
+
+import Algebra.Module.Properties          as ModuleProperties
+import Algebra.Module.Morphism.Structures as MorphismStructures
+
+open import Algebra                   using (CommutativeRing)
+open import Algebra.Module            using (Module)
+open import Algebra.Module.Morphism.Bundles
+open import Axiom.DoubleNegationElimination
+open import Data.Product
+open import Level                     using (Level)
+open import Relation.Binary.Reasoning.MultiSetoid
+open import Relation.Nullary          using (¬_)
+open import Relation.Nullary.Negation using (contraposition)
+
+module LinearMapProperties
+  {r ℓr m ℓm n ℓn : Level}
+  {ring           : CommutativeRing r ℓr}
+  {modA           : Module ring m ℓm}
+  {modB           : Module ring n ℓn}
+  (lm             : LinearMap modA modB)
+  where
```

Comment:
This seems to be an old version of [Algebra.Module.Morphism.ModuleHomomorphism](https://github.com/agda/agda-stdlib/blob/master/src/Algebra/Module/Morphism/ModuleHomomorphism.agda)? Can we remove this file from the PR?